### PR TITLE
Document new Free trial budget

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement
- free trial budget documented
- promotions section split up

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/promotions/free-trial-budget.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.. _free-trial-budget:
+
+=================
+Free trial budget
+=================
+
+This section explains our Free trial budget mechanism. This feature gives the
```
Comment (SStorm) [inline docs/promotions/free-trial-budget.rst:7]: I would perhaps avoid mentioning this - we do give the credits to any new org the user creates, but we don't want them to be creating 10s of orgs to exploit this. Maybe something like "This features gives new users $200 of free credits, to be used with any cluster configuration for one month."?

File: docs/promotions/index.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+offers on CrateDB Cloud database-as-a-service products. Promotions can involve
+discounts of up to 100% on any dimensions of the product: for example, there
+can be promotions on specific `subscription plans`_ or on `scale units`_ (the
+number of nodes in a cluster) within a plan. Promotions may last indefinitely
+or for limited periods of time.
+
+Another promotional mechanism that should be useful for testing purposes or for
+users new to CrateDB Cloud is our Free trial budget system. This provides you
+with a free credit whenever you create a new organization. You can spend this
+credit on any CrateDB Cloud development plan you choose.
```
Comment (SStorm) [inline docs/promotions/index.rst]: Again, would not mention that this is given every time you create a new org ;-) Maybe something like:

```suggestion
CrateDB Cloud also offers new users $200 in free credits upon signup. You can spend these
credits on any CrateDB Cloud cluster configuration you choose.
```

Comment (matkuliak) [reply docs/promotions/index.rst]: Ah, that's probably true. Better to obfuscate that a little bit. Better now?

**Review by SStorm — APPROVED** [review]
:100:

**Pull request merged**